### PR TITLE
Adding "difficult", "truncated", "occluded" attributes when converting to Pascal VOC if they are not present

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Summary

Allowed adding "difficult", "truncated", "occluded" attributes when converting to Pascal VOC if these attributes are not present

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by yasakova-anastasia** [comment]
@zhiltsov-max , do you think we need to do this for all attributes?

**Comment by zhiltsov-max** [comment]
Yes, but I'd also consider changing AC code, or adding an option to convert in "compressed" mode.

File: datumaro/plugins/voc_format/converter.py [hunk 1]
```diff
@@ -236,20 +236,17 @@ def save_subsets(self):
...
+                        ET.SubElement(obj_elem, 'truncated').text = \
+                            '%d' % truncated
+
+                        occluded = _convert_attr('occluded', attr, int, 0)
+                        ET.SubElement(obj_elem, 'occluded').text = \
+                            '%d' % occluded
+
+                        difficult = _convert_attr('difficult', attr, int, 0)
+                        ET.SubElement(obj_elem, 'difficult').text = \
+                            '%d' % difficult
```
Comment (zhiltsov-max) [inline datumaro/plugins/voc_format/converter.py:249]:
```suggestion
                        ET.SubElement(obj_elem, 'truncated').text = \
                            '%d' % _convert_attr('truncated', attr, int, 0)

                        ET.SubElement(obj_elem, 'occluded').text = \
                            '%d' % _convert_attr('occluded', attr, int, 0)

                        ET.SubElement(obj_elem, 'difficult').text = \
                            '%d' % _convert_attr('difficult', attr, int, 0)
```

Comment (zhiltsov-max) [inline datumaro/plugins/voc_format/converter.py:249]:
```suggestion
                        ET.SubElement(obj_elem, 'truncated').text = \
                            '%d' % _convert_attr('truncated', attr, int, 0)
                        ET.SubElement(obj_elem, 'occluded').text = \
                            '%d' % _convert_attr('occluded', attr, int, 0)
                        ET.SubElement(obj_elem, 'difficult').text = \
                            '%d' % _convert_attr('difficult', attr, int, 0)
```

**Pull request merged**